src/test/java/api/loans/agetolost/ScheduledAgeToLostApiTest.java

```diff
@@ -183,19 +186,13 @@ private MultipleJsonRecords getLoanActions() {
   }
 
   private Matcher<JsonObject> hasPatronBillingDate() {
-    return hasPatronBillingDate(overdueLoan);
-  }
-
-  private Matcher<JsonObject> hasPatronBillingDate(IndividualResource loan) {
-    val expectedBillingDate = parse(loan.getJson().getString("dueDate"))
-      // age to lost overdue interval, per default policy
-      .plusMinutes(1)
+    val expectedBillingDate = DateTime.now(UTC)
       // bill patron after age to lost interval, per default policy
       .plusMinutes(5);
 
     return allOf(hasJsonPath("agedToLostDelayedBilling.lostItemHasBeenBilled", false),
       hasJsonPath("agedToLostDelayedBilling.dateLostItemShouldBeBilled",
-        isEquivalentTo(expectedBillingDate)));
+        withinSecondsBefore(Seconds.seconds(1), expectedBillingDate)));
```

Comment:
Given the expected billing date is now dependent upon when the item was aged to lost. Could this check be specific based upon the date the item was aged to lost (i.e. derive the expected date from the aged to lost date)?

Reply:
Make sense, done.